### PR TITLE
Verify artifacts with the sha256 hash passed as an argument.

### PR DESCRIPTION
**Pull request by mitoma**
refs #8

### CODE REVIEW

File: setup/README.md [hunk 1]
```diff
@@ -17,14 +17,16 @@ With all options set:
   with:
     os: linux
     version: v0.1.14
+    sha256sum: 407875b9e2263fb3ca94a6be166cd280f92404f01b1a00989a6deec441635706
```
Comment (itchyny) [inline setup/README.md:20]: This usage config uses the local action?

Comment (mitoma) [reply setup/README.md:20]: @itchyny 
Indeed, that’s strange. I was expecting an action from an external repository, so I’ll fix the usage.

Comment (itchyny) [reply setup/README.md:20]: This is `setup` action, not `exec` action.

File: setup/README.md [hunk 2]
```diff
@@ -13,18 +13,22 @@ Install sver and set the path just by adding the following line to steps.
...
-| ------- | ---------------------------------------- |
-| os      | `linux` or `macos` or `windows`          |
-| version | released sver version. [sver releases][] |
+| name      | value                                    |
+| --------- | ---------------------------------------- |
+| os        | `linux` or `macos` or `windows`          |
+| version   | released sver version. [sver releases][] |
+| sha256sum | sha256sum of artifact zip file           |
+
+Please refer to the `SHASUMS256.txt` included in the [sver release][] for the sha256 hash.
```
Comment (itchyny) [inline setup/README.md:31]:
```suggestion
Please refer to the `SHASUMS256.txt` included in the [sver releases][] for the sha256 hash.
```

**Pull request merged**